api/envoy/api/v2/cds.proto

```diff
@@ -427,6 +427,12 @@ message Cluster {
 
   // Determines how Envoy selects the protocol used to speak to upstream hosts.
   ClusterProtocolSelection protocol_selection = 26;
+
+  // If an upstream host becomes unhealthy (as determined by the coonfigured health checks
+  // or outlier detection), immediately close all connections to the failed host.
+  //
+  // This is currently only supported for connections created by tcp_proxy.
```

Comment:
More specifically it's for any filter that ends up using tcpConnForCluster(), of which I guess tcp_proxy is the only one. Worth calling out? Not sure.

Reply:
I thought about that, but exposing that detail in the docs seems out-of-place.